source/editing_pages/dashboard/contact_form.rst

```diff
@@ -4,16 +4,9 @@ Contact form
 This widget inserts a form that users can use to contact the domain's administrators. It uses reCaptcha to make sure
 anonymous users cannot spam the administrators.
```

Comment:
No line break needed here

Reply:
since this is an RST file, line breaks are not shown in the output. They are here only to avoid having lines that are too long in our code editors.